modules/rdg/src/base/RdgApp.C

```diff
@@ -0,0 +1,52 @@
+#include "RdgApp.h"
+#include "Moose.h"
+#include "AppFactory.h"
+#include "MooseSyntax.h"
+
+template<>
+InputParameters validParams<RdgApp>()
+{
+  InputParameters params = validParams<MooseApp>();
+
+  params.set<bool>("use_legacy_uo_initialization") = false;
+  params.set<bool>("use_legacy_uo_aux_computation") = false;
+  params.set<bool>("use_legacy_output_syntax") = false;
```

Comment:
I would be nice to have these default to `false`, so new stuff does not have to mess with historical garbage...